parser/gjson_extend.go

```diff
@@ -70,9 +70,9 @@ func injectObject(prefix string, result map[string]interface{}, t gjson.Result)
 	t.ForEach(func(key gjson.Result, value gjson.Result) bool {
 		switch value.Type {
 		case gjson.JSON:
-			injectObject(prefix+"_"+key.String(), result, value)
+			injectObject(prefix+"."+key.String(), result, value)
```

Comment:
Hi, It‘s ok to have  `kafkaField`, but why we need to change to separated char from `_` to  `.`?

Reply:
@sundy-li I think some of people will name the json field with underscore like this. That's why I change to separated char from `_` to `.`
```
{
   "data":{
      "level_1": 1,
      "level_2": 2,
      "level_3": 3
   }
}
```